packages/devtools/src/composite.ts

```diff
@@ -541,11 +543,16 @@ export class Composite {
 
   /**
    * Configure the Ceramic node to index the models defined in the composite. An authenticated DID
-   * set as admin in the Ceramic node configuration must be attached to the Ceramic instance.
+   * set as admin in the Ceramic node configuration must be attached to the Ceramic instance, otherwise
+   * an error is thrown.
    */
   async startIndexingOn(ceramic: CeramicApi): Promise<void> {
     const modelIDs = Object.keys(this.#definition.models).map(StreamID.fromString)
-    await ceramic.admin.startIndexingModels(modelIDs)
+    try {
+      await ceramic.admin.startIndexingModels(modelIDs)
+    } catch(e: MissingDIDError) {
```

Comment:
Is this the only type of error that can happen please? What if the node is unreachable and there's a HTTP 404 error instead for example?

Reply:
the purpose here is to make the error clearer specifically for missing did.  other errors will continue to behave as they do now.